src/main/java/net/atos/zac/app/zaken/model/RESTBesluittype.java

```diff
@@ -15,4 +17,6 @@ public class RESTBesluittype {
 
     public String toelichting;
 
+    public List<URI> informatieObjectTypen;
```

Comment:
informatieObjectTypen hernoemen naar informatieobjecttypen

Reply:
Opgelost in opvolgende commit